drm-ffi/drm-sys/src/platforms/freebsd/x86_64/bindings.rs

```diff
@@ -343,7 +343,7 @@ pub struct __max_align_t {
 #[test]
 fn bindgen_test_layout___max_align_t() {
     assert_eq!(
-        ::std::mem::size_of::<__max_align_t>(),
+        ::core::mem::size_of::<__max_align_t>(),
```

Comment:
This diff looks larger than it really is, and basically just consists of `s/std/core` replacements.

There are also the derive impls for `Eq`, `PartialEq`, and `Hash` on three of the `drm_mode_*` structs.

Reply:
@orvij This file is automatically generated by `drm-ffi/drm-sys/build.rs`, and a re-run of that will overwrite all your changes. It looks like this file for FreeBSD hasn't been regenerated in four years when `.use_core()` was only added to the bindgen setup a year ago: 6ea69747903a130349c62b116f1faf27e43d3e47. I suggest you run `cargo b --manifest-path drm-ffi/drm-sys/Cargo.toml --features update_bindings` on FreeBSD and commit the result instead. Note that this build script does not always re-run (it should have been a runnable binary, IMO) and might need a `cargo clean` before.